message_sender/views.py

```diff
@@ -89,6 +90,18 @@ class InboundViewSet(viewsets.ModelViewSet):
                      'content', 'transport_name', 'transport_type',
                      'helper_metadata', 'created_at', 'updated_at',)
 
+    def create(self, request, *args, **kwargs):
+        expect = ["event_type", "in_reply_to"]
+        if set(expect).issubset(request.data.keys()):
+            if request.data['event_type'] == "close":
```

Comment:
This is an inbound message, not an event, so it looks more like 
```{"channel_data": {"session_event": "close"}, "from": "127.0.0.1:52633", "channel_id": "ac7fe54f-9b60-4ea3-9388-3d1692ab4a0e", "timestamp": "2016-11-21 07:56:29.784856", "content": null, "to": "0.0.0.0:9001", "reply_to": null, "message_id": "2d0c7ca5b96d4ddc8c9ad713e0f5ba4a"}``` 
from Junebug, and 
```{"session_event": "close", "from_addr": "127.0.0.1:52633", "transport_name": "ac7fe54f-9b60-4ea3-9388-3d1692ab4a0e", "timestamp": "2016-11-21 07:56:29.784856", "content": null, "to_addr": "0.0.0.0:9001", "in_reply_to": null, "message_id": "2d0c7ca5b96d4ddc8c9ad713e0f5ba4a"}``` 
for a message from vumi.